components/hero-unit.jsx

```diff
@@ -7,11 +7,16 @@ var HeroUnit = React.createClass({
   render: function() {
     return (
```

Comment:
what about a function here to calculate the device DPR and set the appropriate source to a variable that the `hero-unit` class can use?